src/components/ProjectsPage/projects.scss

```diff
@@ -13,46 +14,55 @@
     position: relative;
     display: flex;
     flex-wrap: wrap;
+    align-content: end;
```

Comment:
```suggestion
    align-content: flex-start;
```
To fix this:
![image](https://user-images.githubusercontent.com/13918850/77414047-0b14fb80-6dc9-11ea-92ea-8fc0b2d00480.png)